install/running_install.adoc

```diff
@@ -154,9 +154,11 @@ system can run out of files to open and the playbook will fail.
 
 To run the RPM-based installer:
 
-. Run the *_prequisites.yml_* playbook. This must be run only once before
-deploying a new cluster. Use the following command, specifying `-i` if your
-inventory file located somewhere other than *_/etc/ansible/hosts_*:
+. Run the *_prequisites.yml_* playbook. This playbook modifies the container
+runtimes and installs required software packages, if any. Unless you need to
+modify the container runtimes, run this playbook only one time, before you
+deploy a cluster the first time. Use the following command, specifying `-i`
+if your inventory file is not in the *_/etc/ansible/hosts_* directory:
```

Comment:
`/etc/ansible/hosts` should be a file path, not a directory. The original statement is more accurate.